cnf-certification-test/lifecycle/suite.go

```diff
@@ -372,3 +378,33 @@ func testPodsRecreation(env *provider.TestEnvironment) { //nolint:funlen
 		}
 	}
 }
+
+func testPodRequestsAndLimits(env *provider.TestEnvironment) {
+	var containersMissingRequestsOrLimits []string
+	ginkgo.By("Testing container resource requests and limits")
+
+	// Loop through the containers, looking for containers that are missing requests or limits.
+	// These need to be defined in order to pass.
+	for _, cut := range env.Containers {
+		badContainer := false
+		if len(cut.Data.Resources.Limits) == 0 {
```

Comment:
The requirement states that "Pods must define requests and limits values for **CPU** and **memory**" So, shouldn't this condition also check for the existence of the strings "cpu" and "memory" in the map?

Reply:
Good call, I can get more granular instead of just checking the top level.  Will add.

Reply:
Okay I added checks to make sure that CPU and memory are set.